src/components/create-user.html

```diff
@@ -21,45 +21,57 @@ <h1>Create User</h1>
         <legend id="createNewUser">New Profile</legend>
         <div class="form-group">
           <label for="firstName" id="inputFirstName">First Name</label>
-          <input type="text" class="form-control"  aria-describedby="createNewUser inputFirstName"
-            placeholder="Enter first name">
-
+          <input type="text" class="form-control" aria-describedby="createNewUser inputFirstName"
+            placeholder="Enter first name" value="{{newUserData.firstName::input}}">
         </div>
 
         <div class="form-group">
           <label for="lastName" id="inputLastName">Last Name</label>
-          <input type="text" class="form-control"  aria-describedby="createNewUser inputLastName"
-            placeholder="Enter last name">
-
+          <input type="text" class="form-control" aria-describedby="createNewUser inputLastName"
+            placeholder="Enter last name" value = "{{newUserData.lastName::input}}">
         </div>
 
         <div class="form-group">
-          <label for="phone" id="inputPhone"> Phone</label>
-          <input type="text" class="form-control"  aria-describedby="createNewUser inputPhone"
-            placeholder="Enter phone">
-
+          <label for="phone" id="inputPhone">Phone</label>
+          <input type="text" class="form-control" aria-describedby="createNewUser inputPhone" placeholder="Enter phone" value="{{newUserData.phone::input}}">
         </div>
 
         <div class="form-group">
           <label for="email" id="inputEmail">Email</label>
           <input type="email" class="form-control" aria-describedby="createNewUser inputEmail"
-            placeholder="Enter email">
-
+            placeholder="Enter email" value="{{newUserData.email::input}}">
         </div>
 
-        <button type="submit" class="btn btn-primary">Submit</button>
+        <button type="submit" class="btn btn-primary" on-click="subBut">Submit</button>
 
       </fieldset>
     </form>
   </template>
 
-
   <script type="module">
     import { Element as PolymerElement } from '@banno/polymer/polymer-element.js';
+    import { DataStructureCreate } from '../api/data-structure-creator.js';
+
     class CreateUser extends PolymerElement {
       static get is() { return 'create-user'; }
+      
+      static get properties() {
+        return {
+          storage: Object,
+          newUserData: {
+            type: Object,
+            value() {
+              return {};
+            }
+          }
+        }
+      }
+
+      subBut(){
+        console.log(this.storage.createUser(this.newUserData));
```

Comment:
there a reason you're running this in a console log? I understand it returns the new user when successful, but I'd think you'd want to do something like
```suggestion
       const returnedUser = this.storage.createUser(this.newUserData));
       console.log(returnedUser);
```

That said, I'm not super sure why you'd want to log the user at all.

Reply:
No intention here, a mistake  - I missed it on my console.log purge